src/short_id.rs

```diff
@@ -20,14 +20,21 @@ use bitcoin::{BlockHash, Txid};
 
 #[derive(Copy, Clone, Debug, Display, PartialEq, Eq)]
 #[display(Debug)]
+/// Errors from descriptor validation and parsing
 pub enum Error {
+    /// block height invalid error
```

Comment:
here and below we can remove `error` from the end, since it will be displayed as `Error: <message>`

```suggestion
    /// invalid block height in bitcoin ShortId
```

Reply:
Makes sense, thanks for suggestion, still collecting some Rust best practices :slightly_smiling_face: